go-controller/pkg/ovn/egressgw.go

```diff
@@ -178,11 +193,11 @@ func (oc *Controller) addPodExternalGW(pod *kapi.Pod) error {
 // addPodExternalGWForNamespace handles adding routes to all pods in that namespace for a pod GW
 func (oc *Controller) addPodExternalGWForNamespace(namespace string, pod *kapi.Pod, egress gatewayInfo) error {
 	var gws string
-	for _, ip := range egress.gws {
+	for _, ip := range egress.gws.UnsortedList() {
```

Comment:
@tssurya since this is now a string list, we can use `gws := strings.Join(egress.gws.UnsortedList(), ",")` maybe?